.github/workflows/keycloak-scalability-benchmark.yml

```diff
@@ -143,8 +144,50 @@ jobs:
           path: keycloak-benchmark/results
           retention-days: 5
 
+      - name: Archive Summary
+        uses: actions/upload-artifact@v3
+        with:
+          name: summary
+          path: ${{ steps.kcb.outputs.kcb_result }}
+          retention-days: 5
+
       - name: Delete Keycloak deployment
         if: ${{ !inputs.skipDeleteProject }}
         uses: ./.github/actions/keycloak-delete-deployment
         with:
           project: ${{ env.PROJECT }}
+
+  archive:
+    name: Commit results to Git repository
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    needs:
+      - run
+    steps:
+
+      - name: Checkout repository for results
+        uses: actions/checkout@v3
+        with:
+          ref: 'result-data'
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: summary
+
+      - name: Commit result-summary
+        shell: bash
+        env:
+          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
would this create additional github bot commit noise ? or is this against a non-main branch line ?

Reply:
@kami619 - all results go to the branch 'result-data', so there is no Git commit noise in the main branch. Eventually we might move it to its own repository, but it would make things a bit more complicated as it would need a personal access token for this, and it would be harder to test on forks.

Reply:
cool. thank you.